recipes/collector-sidecar/recipe.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../tools'
+
+class GraylogSidecar < FPM::Cookery::Recipe
+  description 'Graylog collector sidecar'
+
+  name     'graylog-sidecar'
+  version  data.version
+  revision data.revision
+  homepage 'https://graylog.org'
+  arch     'amd64'
+
+  source data.source
+  sha256 data.sha256
+
+  maintainer 'Graylog, Inc. <hello@graylog.org>'
+  vendor     'graylog'
+  license    'GPLv3'
+
+  config_files '/etc/graylog/sidecar/sidecar.yml'
+
+  fpm_attributes rpm_os: 'linux'
+
+  def build
+  end
+
+  def install
+    bin.install "#{version}/linux/amd64/graylog-sidecar"
+    etc('graylog/sidecar').install '../../sidecar-example.yml', 'sidecar.yml'
```

Comment:
See below for the `workdir` helper comment.

Reply:
fixed.